test/e2e/specs/dev_spec.js

```diff
@@ -39,10 +39,15 @@ describe('Dev Mode flows', function () {
     util.loadTestPage('basic-dev');
   });
 
+  afterEach(function () {
+    // Logout of Okta session
+    browser.get('{{{WIDGET_TEST_SERVER}}}/login/signout');
```

Comment:
nit: Consider adding an expectation here to verify we're back on the primary auth page to confirm logging out was successful.